static/app/components/sidebar/sidebarDropdown/index.tsx

```diff
@@ -31,9 +31,19 @@ type Props = Pick<CommonSidebarProps, 'orientation' | 'collapsed'> & {
   org: Organization;
   user: User;
   config: Config;
+  //set to true for deactivated members so we can hide links they cannot access
+  hideInternalLinks?: boolean;
```

Comment:
Can you use jsdoc style comments so we get this in the typescript hover for this prop

```
/**
 * set to true for deactivated members so we can hide links they cannot access
 */
```

Reply:
Also IMO I would change the comment here to be less specific to disabled members, since you've already done that for the name of this prop. Just say what it does.

Reply:
Also is `internal` the right word here?

Reply:
Do single line comments not show?

Reply:
They do not

![image](https://user-images.githubusercontent.com/1421724/120381529-66008780-c2d7-11eb-8b47-378301c98e7f.png)
![image](https://user-images.githubusercontent.com/1421724/120381545-6b5dd200-c2d7-11eb-9b2e-c8d759a0a8f5.png)

Reply:
updated comment